### PR TITLE
Documenting deprecation of the system_profile plugin

### PR DESCRIPTION
**Pull request by mjingle**
Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

### Description
Documents upcoming depreciation of system_profile plugin in Ohai

### Definition of Done

When @tas50 says it's okay.

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by mjingle** [comment]
@tas50 I noticed that system_profile plugin is listed as "system_profiler.rb" in the list of default plugins available. I assume those are the same thing?

**Review by tas50 — CHANGES_REQUESTED** [review]
Instead of this we'll actually want to create an Ohai deprecations number and create a deprecations page for the removal of the plugin. 

Here's a sample removal deprecation:

https://docs.chef.io/deprecations_ohai_ipscopes.html

The deprecation would go here:

https://docs.chef.io/chef_deprecations_client.html

File: chef_master/source/ohai.rst [hunk 1]
```diff
@@ -218,6 +218,8 @@ For attributes that contain slashes (``/``) within the attribute value, such as
 
 Default Plugins
 =====================================================
+.. warning:: The system_profile plugin will be removed from Chef/Ohai 15 in April 2019. This plugin does not correctly return data on modern Mac systems. Additionally the same data is provided by the hardware plugin, which has a format that is simpler to consume. Removing this plugin will reduce Ohai return by ~3 seconds and greatly reduce the size of the node object on the Chef server.
```
Comment (tas50) [inline chef_master/source/ohai.rst:221]: We usually just create a deprecation page on the deprecations page for these. There's too many to track with warnings.

Comment (mjingle) [reply chef_master/source/ohai.rst:221]: @tas50 Is there a certain code that needs to be used for this on the depreciations page?

**Pull request closed without merging**